src/base/index.spec.tsx

```diff
@@ -252,20 +252,17 @@ describe('Portal', () => {
         <>
           <Portal />
           <Button onClick={() => setOpen(true)}>Open</Button>
-          {open && (
```

Comment:
I've generally found conditionally rendering dialogs in RMWC to be pretty flaky on React 18 - in general I just let the control state decide if it should be visible or not, but happy to discuss!